taipy/common/config/config.py

```diff
@@ -25,6 +25,13 @@
 from .section import Section
 from .unique_section import UniqueSection
 
+class RestConfig:
+    def __init__(self):
+        self.port = 5000  # Default port
+        self.host = "127.0.0.1"  # Default host
+        self.use_https = False
+        self.ssl_cert = None
+        self.ssl_key = None
```

Comment:
This is obsolete

Reply:
do i delete this class then?

Reply:
You have the RestSection class in the `rest_config.py` file, no?
Or you mean the `RestSection` and `RestConfig` have different purpose?

Reply:
No I think they are performing the same task, just assigning values to these variables